Backpack/Font/Classes/Generated/BPKFont.m

```diff
@@ -180,6 +226,133 @@ + (UIFont *)fontForStyle:(BPKFontStyle)style fontManager:(BPKFontManager *)fontM
     }
 }
 
++ (NSNumber *)letterSpacingForStyle:(BPKFontStyle)style {
+    switch (style) {
+        case BPKFontStyleTextBase: 
```

Comment:
Are these the legacy letter spacings? I think we need to remove these as they no longer apply (only the hero ones are correct)

Reply:
Yes, those are the legacy ones.
Should I remove these and only leave the new ones?

Reply:
yes let's remove them